src/main/scala/org/phenoscape/kb/PresenceAbsenceOfStructure.scala

```diff
@@ -81,11 +84,12 @@ object PresenceAbsenceOfStructure {
   def presenceAbsenceMatrix(mainEntityClass: OWLClassExpression,
                             taxonClass: OWLClassExpression,
```

Comment:
Need to accept a list of taxa and a list of entities, if not query for subclasses.